spec.md

```diff
@@ -267,13 +269,42 @@ Where the decoded payload of the request parameter is as follows:
     "x":"7KEKZa5xJPh7WVqHJyUpb2MgEe3nA8Rk7eUlXsmBl-M",
     "y":"3zIgl_ml4RhapyEm5J7lvU-4f5jiBvZr4KgxUjEhl9o"
   },
-  "credential_format": "w3cvc-jwt",
+  "credential_format": "w3cvc-jsonld",
   "nonce": "43747d5962a5",
   "iat": 1591069056,
   "exp": 1591069556
 }
 ```
 
+## Down Scoped Credential Request
+The `claims` property may be used when requesting a Credential in order to reduce the claims returned by the CI. 
+
+The following is a non-normative example of the decoded payload of the request parameter of a down scoped Signed Credential request.
+
+```
+{
+  "aud": "https://issuer.example.com",
+  "iss": "https://wallet.example.com",
+  "sub": "urn:uuid:dc000c79-6aa3-45f2-9527-43747d5962a5",
+  "sub_jwk" : {
+    "crv":"secp256k1",
+    "kid":"YkDpvGNsch2lFBf6p8u3",
+    "kty":"EC",
+    "x":"7KEKZa5xJPh7WVqHJyUpb2MgEe3nA8Rk7eUlXsmBl-M",
+    "y":"3zIgl_ml4RhapyEm5J7lvU-4f5jiBvZr4KgxUjEhl9o"
+  },
+  "credential_format": "w3cvc-jwt",
+  "nonce": "43747d5962a5",
+  "iat": 1591069056,
+  "exp": 1591069556,
+  "claims": {
```

Comment:
Should probably also make it clear that the syntax of the claims request parameter is inherited from the relevant section of OpenID Connect Core?

Reply:
yes good point, let me add a sentence here.

Reply:
updated in latest commit adding ref to section 5.5 in the intro of the claims property.

Reply:
Actually now that I review this, the first level in the claims object is actually the location in which the claims will appear e.g `id_token` or `userinfo_endpoint` so if we dont want that we will need to define our own parameter syntax

Reply:
right right, yes the nesting under the top level `claims.credential` felt redundant but may be easier and then compliant with existing syntax? Versus rolling our own definition of `claims` ?

Reply:
ping @tplooker what do ya think on this one? :)

Reply:
Sorry missed this? TBH I like not having the redundant nesting, I just think we have to cleanup the language to clarify the syntaxes relationship to that seen in OpenID Connect Core

Reply:
E.g they share the syntax on how to request claims but do not require the 'location' (e.g id_token/user_info/credential) abstraction

Reply:
Sounds good to me, I prefer that as well. 
Update incoming...